packages/models/src/errors.ts

```diff
@@ -18,7 +18,7 @@ export class ApiError<T> extends Error {
     detail: string;
     correlationId?: string;
   }) {
-    super();
+    super(detail);
```

Comment:
```suggestion
    super();
```
This shouldn't be needed.

Reply:
I think it's needed. Indeed, without `super(detail)` we have this behaviour:
```
expect(promiseThatThrowsAttributeDuplicate).rejects.toThrowError(attributeDuplicate)
// this test passes

expect(promiseThatThrowsOperationForbidden).rejects.toThrowError(attributeDuplicate)
// this test passes too, but it shouldn't